trace/client_test.go

```diff
@@ -0,0 +1,47 @@
+package trace
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type server struct {
+	LastSpanID SpanID
+}
+
+func (s server) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	span := ParseSpanID(r)
+	fmt.Fprintf(rw, "%#v", span)
+}
+
+func TestClient(t *testing.T) {
```

Comment:
What does this test case actually tests? 

Please use more descriptive variable and test names. It took me a few minutes to make sense out of this test and I'm still not not sure I fully get it. Maybe comments could help as well.